### PR TITLE
fix(settings): Changed modal footer background color for night mode

### LINKED ISSUE #16280: Delete Progress/Brownie Points - Confirmation Popup - cannot read text in dark mode
#### Challenge Name
My Profile -> Update My Settings -> Reset Progress/Brownie Points

#### Issue Description

The confirmation popup window, the text is not readable.
Either the background is too dark because its in dark mode
or the buttons should be filled in and the text be in white

#### Browser Information

* Browser Name, Version:  
* Operating System: FireFox 57.0 (64-bit)  and Chrome Version 63.0.3239.84 (Official Build) (64-bit)
* Mobile, Desktop, or Tablet: Laptop Windows 10 Professional 64-bit

#### Your Code

```js



```
#### Screenshot

image

### LINKED ISSUE #16282: Delete FCC Account - Confirmation Text Popup - cannot read in Dark Mode
#### Challenge Name

My Profile -> Update My Settings -> Reset Progress/Brownie Points

#### Issue Description

The confirmation popup window, the text is not readable.
Either the background is too dark because its in dark mode
or the buttons should be filled in and the text be in white

#### Browser Information

* Browser Name, Version:  
* Operating System: FireFox 57.0 (64-bit)  and Chrome Version 63.0.3239.84 (Official Build) (64-bit)
* Mobile, Desktop, or Tablet: Laptop Windows 10 Professional 64-bit

#### Your Code

```js



```
#### Screenshot

image

### PR DESCRIPTION
**Pull request by vkWeb**
### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

### Checklist:

- [x] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #16282
Closes #16280 

### Description

**Before:**
The options aren't accessible.
capture

**After:**
I changed the background color of the modal footer to `#E0E0E0` because changing text or border color wasn't a good idea. 
capture

### CODE REVIEW

**Comment by vkWeb** [comment]
@raisedadead PR Updated as per your request.

File: client/less/main.less [hunk 1]
```diff
@@ -840,6 +840,9 @@ code {
     .modal-footer {
       border: none;
     }
+    .modal-footer {
```
Comment (raisedadead) [inline client/less/main.less]: Why are using this selector again?

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by raisedadead** [comment]
Also, after taking a second look, I guess we would want the labels to be white

For example like so:
image

File: client/less/main.less (hunk 1 above)
Comment (vkWeb) [reply client/less/main.less]: Do you want me to put this into: 
`.modal-header, .modal-footer{}`? 
Actually, putting values into the above selector will change the background of both the header and footer.

**Comment by vkWeb** [comment]
@raisedadead I'll look into that and will update my PR soon.

**Comment by raisedadead** [comment]
hey @vkWeb how is it going? would you need any help with this?

**Comment by vkWeb** [comment]
@raisedadead I looked into the change you're requesting but labels are already white so what change are you talking about? 
Please elaborate.

**Comment by raisedadead** [comment]
I am referring to the change that you have already made, to the footer. It should be reverted and the labels on the buttons in the modal should be white.

**Comment by BerkeleyTrue** [comment]
@vkWeb To clarify, the buttons themselves should have a different background. The change you made changes the background of the entire footer.

**Comment by vkWeb** [comment]
@raisedadead @BerkeleyTrue Okay - I got it. You want something like this...?: 
If yes, this looks better 👍. I'll implement this change if it's okay.    

capture

**Comment by BerkeleyTrue** [comment]
@vkWeb yes, that looks great.

**Comment by vkWeb** [comment]
@BerkeleyTrue I'll update my PR ASAP.

**Comment by vkWeb** [comment]
@raisedadead, @BerkeleyTrue I have updated the PR as per your request. I'll be waiting for your review.

**Comment by BerkeleyTrue** [comment]
@vkWeb thanks!

Happy Coding

**Comment by vkWeb** [comment]
@BerkeleyTrue Thanks for the merge. 😄

**Pull request merged**